src/data-sources/airtable/index.ts

```diff
@@ -45,17 +47,20 @@ export class AirtableIntrospector extends DataSourceIntrospector {
 
     const {
       data: {
-        records: [record],
+        records: rows,
       },
     } = await this.client.get<{ records: AirtableRecord[] }>(`/${baseId}/${tableId}`, {
       headers: {
-        maxRecords: 1,
+        maxRecords: introspectionDepth,
```

Comment:
hmm is this the depth or length? Perhaps there's another name we can give it like `sampleSize`?